test/recommend/test_recommend_pipeline.py

```diff
@@ -255,3 +255,35 @@ def test_find_start_end_line_in_class(self):
             start, end = find_start_and_end_lines(method)
             self.assertEqual(start, 11)
             self.assertEqual(end, 59)
+
+    def test_find_annotation_by_class_declaration(self):
+        file = Path(self.cur_file_dir, 'annotations/ClassAnnotations.java')
+        with open(file, 'r', encoding='utf-8') as f:
+            tree = javalang.parse.parse(f.read())
+            classes_with_annonations = find_annotation_by_node_type(tree, javalang.tree.ClassDeclaration)
+            pattern_found = list(classes_with_annonations.values())[0][0]
```

Comment:
can we check against all found annotations?

Reply:
this test is about class declaration only